package.json

```diff
@@ -8,7 +8,7 @@
     "prepare": "husky",
     "watch-ts": "tsc -w",
     "build": "node esbuild/esbuild.config.js --build",
-    "start": "node $NODE_OPTIONS dist/server.js | bunyan -o short",
+    "start": "set -a && source .env && node $NODE_OPTIONS dist/server.js | bunyan -o short",
```

Comment:
This should not be included in the start action as it is used by the container and docker compose which don't use .env for the environment variables. Use start:dev instead. 
The app uses the dotenv express module so it should load automatically